README.md

```diff
@@ -166,12 +166,11 @@ Anchors are great for jumping to information farther down a page.
 
 #### Buttons vs.Links vs. Anchors
 ##### The semantic question of link vs button:
- * A button invokes an action: Save, Cancel, Exterminate.
-  * Activated with the `Space` or `Enter` key. Screen reader users may assume this is a form submission and be slightly more trepidacious about activating it than a link.
-  * Identified in screen readers as a button, and included in the list of form controls.
- * A link takes you somewhere (navigates).
-  * Activated with the `Enter` key.
-  * Identified in screen readers as a link, and included in the list of page links.
+| **Buttons** | **Anchors** |
```

Comment:
I think `Anchors` should be `Links`.

Reply:
Done!

Reply:
@estebanz01 I commented below that it needs to be slit into separate columns for "Link" and "Anchor".